docs/source/demo.rst

```diff
@@ -95,7 +95,7 @@ equivalent command using:
 .. code-block:: bash
 
     $ source ga4gh-env/bin/activate
-    (ga4gh-env) $ ga4gh_client readgroupsets-search --datasetIds XXXX http://localhost:8000/current
+    (ga4gh-env) $ ga4gh_client readgroupsets-search --dataset_ids XXXX http://localhost:8000/current
```

Comment:
I agree that command line arguments should be hyphenated - using underscores looks weird - but I was concerned that the parameter names (sans leading '--') might be used as object attribute names.  Danny, does your +1 mean you agree with Jerome (deal with it in a follow-up), or the opposite?